POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Services/ActivationService.cs

```diff
@@ -21,6 +25,88 @@ internal class ActivationService
 
         private object _lastActivationArgs;
 
+        private void newNotification()
+        {
+            
+            // In a real app, these would be initialized with actual data
+            string title = "Power Toys";
+            string content = "Power Toys is installed";
+            string image = "../Assets/Logo.png";
+            string logo = "../Assets/MiniLogo.png";
```

Comment:
Do these images scale correctly across different screen resolutions

Reply:
Discussed offline, notifications dont appear to use copies of different resolutions